tests/data_context/store/test_datasource_store.py

```diff
@@ -48,6 +48,42 @@ def fake_datasource_name() -> str:
     return "my_first_datasource"
 
 
+@pytest.fixture
```

Comment:
I've seen us make fixtures in other places, including this file, so I stuck with that pattern. Probably cleaner to just make top-level constants, unless there's something I'm missing. I'll leave it to the reviewer to make the call.